### PR TITLE
fixed focusing back on county selector (on dashboard)

### PR DESCRIPTION
**Pull request by ckiss**
[ref](https://github.com/project-vagabond/covid-green-app/issues/193)

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -97,7 +97,7 @@ PODS:
     - nanopb/encode (= 1.30905.0)
```
Comment (floridemai) [inline ios/Podfile.lock:85]: Should you update yarn lock?

**Review by floridemai — CHANGES_REQUESTED** [review]
On dashboard after I registered I opened the county selector and tapped on X. The focus was on Share after that. This is not correct. The same thing happened when I chose a county.

**Comment by ckiss** [comment]
@floridemai was this on iphone? seems to work fine on my Android both when I close from X or choosing a country.

File: yarn.lock [hunk 2]
```diff
@@ -10,18 +10,18 @@
     "@babel/highlight" "^7.10.4"
```
Comment (ckiss) [inline yarn.lock:10]: yarn lock is updated @floridemai

**Comment by floridemai** [comment]
> @floridemai was this on iphone? seems to work fine on my Android both when I close from X or choosing a country.

Yes, it was on iOS, but I think we miss the code where we set the focus ourselves when the value changes or on close.

**Review by floridemai — APPROVED** [review]
Works. Thank you

**Pull request merged**